src/app/public/modules/text-expand-repeater/text-expand-repeater.component.ts

```diff
@@ -40,15 +40,25 @@ let nextId = 0;
   ]
 })
 export class SkyTextExpandRepeaterComponent implements AfterViewInit, OnChanges {
-  @Input()
-  public maxItems: number;
 
+  /**
+   * Specifies the data to truncate.
+   */
   @Input()
   public data: any[];
 
+  /**
+   * Specifies a template for items in the list.
+   */
   @Input()
   public itemTemplate: TemplateRef<any>;
 
+  /**
+   * Specifies the number of items to display before truncating the list.
```

Comment:
In the current docs, this is marked as required. Should we add `@required` here?

Reply:
Resolving as per feedback in the `itemTemplate` comment.